### PR TITLE
Added ServiceFilterAttributeBuilder and TypeFilterAttributeBuilder

### PR DESCRIPTION
**Pull request by IvayloGoranov**
#34

### CODE REVIEW

File: src/MyTested.AspNetCore.Mvc.Controllers.Attributes/Builders/Attributes/ServiceFilterAttributeTestBuilder.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ServiceFilterAttributeTestBuilder"/> class.
+        /// </summary>
+        /// <param name="testContext"><see cref="ComponentTestContext"/> containing data about the currently executed assertion chain.</param>
+        /// <param name="failedValidationAction">Action to call in case of failed validation.</param>
+        public ServiceFilterAttributeTestBuilder(
+            ComponentTestContext testContext,
+            Action<string, string> failedValidationAction)
+            : base(testContext, nameof(ServiceFilterAttribute), failedValidationAction)
+        => this.Attribute = new ServiceFilterAttribute(typeof(object));
```
Comment (ivaylokenov) [inline src/MyTested.AspNetCore.Mvc.Controllers.Attributes/Builders/Attributes/ServiceFilterAttributeTestBuilder.cs]: Add one extra tab here.

Comment (IvayloGoranov) [reply src/MyTested.AspNetCore.Mvc.Controllers.Attributes/Builders/Attributes/ServiceFilterAttributeTestBuilder.cs]: Done.

File: src/MyTested.AspNetCore.Mvc.Controllers.Attributes/Builders/Attributes/TypeFilterAttributeTestBuilder.cs [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TypeFilterAttributeTestBuilder"/> class.
+        /// </summary>
+        /// <param name="testContext"><see cref="ComponentTestContext"/> containing data about the currently executed assertion chain.</param>
+        /// <param name="failedValidationAction">Action to call in case of failed validation.</param>
+        public TypeFilterAttributeTestBuilder(
+            ComponentTestContext testContext,
+            Action<string, string> failedValidationAction)
+            : base(testContext, nameof(TypeFilterAttribute), failedValidationAction)
+        => this.Attribute = new TypeFilterAttribute(typeof(object));
```
[the remark this answers is not in the document]
Comment (IvayloGoranov) [reply src/MyTested.AspNetCore.Mvc.Controllers.Attributes/Builders/Attributes/TypeFilterAttributeTestBuilder.cs]: Done.

File: test/MyTested.AspNetCore.Mvc.Test.Setups/Controllers/ApiController.cs [hunk 3]
```diff
@@ -2,12 +2,15 @@
 {
     using Microsoft.AspNetCore.Mvc;
     using Models;
+    using ActionFilters;
```
[the remark this answers is not in the document]
Comment (IvayloGoranov) [reply test/MyTested.AspNetCore.Mvc.Test.Setups/Controllers/ApiController.cs]: Done.

**Pull request merged**